### DISCUSSION TITLE
What's the timeline for 0.83 release?

### DISCUSSION THREAD

**Discussion by yxhua** [mirromutth/r2dbc-mysql#193] [General]
I encountered issue with Cannot encode value of type 'class java.sql.Timestamp'. Since we are connecting to GCP cloudSQL, can't apply the custom converter easily. Anyone can point out some ideas?

**Comment by btakeya** [comment]
I think comments on https://github.com/mirromutth/r2dbc-mysql/issues/151 might help `Timestamp` issue. :)

**Comment by vasilemarian** [comment]
Hi @btakeya , @mirromutth, do you have a schedule when will you release the next version of the jar? 
I would really need a change related to creating custom codecs which we cannot do right now due to the access level for `Codec` class being `package private`.
